config.js

```diff
@@ -1126,6 +1126,9 @@ var config = {
     // the menu has option to flip the locally seen video for local presentations
     // disableLocalVideoFlip: false,
 
+    // Hide the possibility to set and enter a lobby password.
+    // disableLobbyPassword: false,
```

Comment:
We have a bunch of lobby options scattered around... can I please ask you to create a "lobby { ... }" config section to rule them all?